helm/ph-ee-engine-oaf/templates/operations-app.yml

```diff
@@ -35,9 +35,18 @@ spec:
               memory: "512M"
           env:
             - name: "FINERACT_DATASOURCE_CORE_USERNAME"
-              value: "mifos"
+              value: "{{ .Values.operations_app.datasource.username }}" 
```

Comment:
Isn't the same changes there in message gateway?

Reply:
I have added for message gateway also, now check

Reply:
https://github.com/fynarfin/ph-ee-env-labs/pull/26